### ISSUE TITLE
Improve Dasharo docs Dasharo Test Specification navigation bar

### ISSUE THREAD

**Issue by philipandag** [Dasharo/dasharo-issues#1005]
### The problem you're addressing (if any)

The Dasharo Test Specification on docs.dasharo.com is hard to navigate.  

### Describe the solution you'd like

The test suite names in navigation bar should contain the ID and the suite names should be the same as in the test matrix. If not that then at least the suites should sorted alphabetically to allow finding the test suite specification quicker.

### Where is the value to a user, and who might that user be?

Mostly testers doing manual tests would be able to find a test suite specification much quicker just by searching from the ID or name from the test matrix, or at least by bisecting if the names would be sorted.

### Describe alternatives you've considered

If someone is not familiar with the specification the best way is to just look at every suite one by one until you find the test ID you have been looking for.

**Comment by mkopec** [comment]
Indeed, I submitted a PR sorting the tests, but it probably needs to be redone for the latest docs revision: https://github.com/Dasharo/docs/pull/741/files

I wouldn't even try rebasing, just re-do everything from scratch. You sort lines alphabetically in vim using `:sort`

**Comment by philipandag** [comment]
My intention here is to be able to be able immediately go to the correct test suite in documentation by looking on the output of robot, example:

```
==============================================================================
Wifi-Bluetooth-Support                                                        
==============================================================================
WLE003.001 Bluetooth scanning (Ubuntu) :: Check whether the Blueto... ..Remember to test all variants of wireless cards.
[ WARN ] Remember to test all variants of wireless cards.                     
WLE003.001 Bluetooth scanning (Ubuntu) :: Check whether the Blueto... | FAIL |
'Failed to set power on: org.bluez.Error.Failed' does not contain 'Changing power on succeeded'
------------------------------------------------------------------------------
Wifi-Bluetooth-Support                                                | FAIL |
1 test, 0 passed, 1 failed
==============================================================================
```
But the name of the test suite in documentation's nav bar is `Wi-Fi and Bluetooth support ` which does not appear in the output so finding it is not as straightforward as, in my opinion, it should be.

If the suite names in navigation bar contained the test ID it would be just a matter of searching for the ID. Sorting is not necessary if I would be able to `ctrl+f` and find the document quickly.

**Comment by philipandag** [comment]
A practical example. I would like to run `SMW001.001` test case, in the matrix it is described as "Wi-fi connection after cold boot (Ubuntu)" but nothing like this can be found in `dasharo-compatiblity/` directory. I had to use `grep -r` to search the test module looking for "SMW" and it turns out the file is called `m2-wifi.robot`... and it is not in the compatibility module like the matrix suggests 
![image](https://github.com/user-attachments/assets/2559ffd0-3baa-4859-a789-92b2d7d9a059)
but in the stability module. So looking for the test documentation I now had to switch to the stability module and now knowing, that the test suite file name is m2-wifi I could find the documentation, but by looking just looking at matrix I would not be able to find the documentation quickly.

**Comment by philipandag** [comment]
The HBN Hibernation test suite documentation is missing or unintuitive and hard to find

**Comment by mkopec** [comment]
Fixed by https://github.com/Dasharo/docs/pull/955
